src/001._two_sum.md

```diff
@@ -0,0 +1,72 @@
+## Two Sum
+
+### Description
+
+```Python
+Given an array of integers, return indices of the two numbers such that they add up to a specific target.
+
+You may assume that each input would have exactly one solution, and you may not use the same element twice.
+
+Example:
+
+Given nums = [2, 7, 11, 15], target = 9,
+
+Because nums[0] + nums[1] = 2 + 7 = 9,
+return [0, 1].
+```
+
+### Solutions
+
+#### Approach 1: Double loop
+
+* Time complexity : O(n^2)
+* Space complexity : O(1)
+
+```Python
+class Solution(object):
+    def twoSum(self, nums, target):
+        """
+        Given an array of integers, return indices of the two numbers such that they add up to a specific target.
+
+        Args:
+            nums(List[int]): the name of array of integers
+            target(int): the value that we assign
+        
+        Returns:
+            the indices of the two numbers from the array
+        """    
+        for i in range(len(nums)):
+            for j in range(i+1, len(nums)):
+                if nums[i] + nums[j] == target:
+                    return [i, j]
+                else:
+                    return None
```

Comment:
The return None statement should not appear here. The function should return None only after the outer loop is completely executed.